crates/view/src/planner.rs

```diff
@@ -549,7 +549,7 @@ impl<R: RngCore + CryptoRng> Planner<R> {
 
     /// Add spends and change outputs as required to balance the transaction, using the view service
     /// provided to supply the notes and other information.
-    pub async fn plan<V: ViewClient>(
+    pub async fn plan<V: ViewClient + ?Sized>(
```

Comment:
Normally I'd quibble about the API change, but given that we're already deep in consensus-breaking LQT territory and tagging, v2 alphas, let's go :+1:

Reply:
This actually makes the API more *lax*. By default, Rust assumes that the generic type *is* sized. This modifies the function to *not assume* that the generic type is sized, allowing more values to be passed in to this function. I think this is, if anything, an omission in the original implementation, because we never actually ran into consumers that implemented the trait despite not being sized (which you do when working with dyn Trait pretty easily)